storage/migrations/00_consensus.up.sql

```diff
@@ -74,12 +67,11 @@ CREATE TABLE chain.transactions
   PRIMARY KEY (block, tx_index)
 );
 -- Queries by sender and/or tx_hash are available via the API.
-CREATE INDEX ix_transactions_sender ON chain.transactions (sender); -- removed in 18_consensus_tx_idxs.up.sql
 --`sender` is a possible external API parameter; `block` lets us efficiently retrieve the most recent N txs with a given method.
--- CREATE INDEX ix_transactions_sender_block ON chain.transactions (sender, block); -- added in 18_consensus_tx_idxs.up.sql
+CREATE INDEX ix_transactions_sender_block ON chain.transactions (sender, block);
 CREATE INDEX ix_transactions_tx_hash ON chain.transactions (tx_hash);
 --`method` is a possible external API parameter; `block` lets us efficiently retrieve the most recent N txs with a given method.
--- CREATE INDEX ix_transactions_method_block ON chain.transactions (method, block); -- added in 18_consensus_tx_idxs.up.sql
+CREATE INDEX ix_transactions_method_height ON chain.transactions (method, block);
```

Comment:
I should have named this index `ix_transactions_method_block` (as it was here in the comment).